### DISCUSSION TITLE
Error al Subir imágenes al formulario

### DISCUSSION THREAD

**Discussion by RomeoRuiz** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#289] [Q&A]
Hola buenas noches compañeros, tengo este problema al subir las imágenes para rellenar el formulario de la entrega, alguien mas tiene este problema? Ya probé en varios navegadores y da el mismo problema.
![Screenshot 2022-03-06 at 23-34-46 Entrega HTML LaunchX](https://user-images.githubusercontent.com/99459578/156974272-d4c79737-3970-45e0-a043-2ef8f11f718d.png)

**Comment by GloriaCoralCerecedo** [comment]
A mí me pasó pero luego la página se recargo sola y me dejó

**Comment by SebbyLebby** [comment]
Les sigue abriendo el enlace? :(

**Reply by JuanSalvador73** [reply]
Lo mandé antes de las 12, media hora antes. Si no te abre ciérralo e inténtalo de nuevo.

**Comment by JuanSalvador73** [comment]
Checa el tamaño y formato de imagen, a veces eso puede marchar error. Que no sean muy grandes o que tengan extensiones raras.

**Reply by RomeoRuiz** [reply]
Ya probé de todo, cambiar resolución de las imágenes, formatos, reiniciar el modem, reiniciar la compu, desde el celular, con datos, etc, etc, (Todo esto antes de las 12) y no me dejo con nada, me sigue abriendo el formulario pero sigue el error
![Captura de pantalla (error)](https://user-images.githubusercontent.com/99459578/156980526-48ca9967-ae62-44cd-818c-f1eddc801f7b.png)

**Comment by JulioMezaV** [comment]
-revisa que si estes desde tu cuenta de innovaccion 
-a mi me dio un error cuando trate de subir mas de un archivo al mismo tiempo tuve que cargarlos de uno en uno

**Comment by JosePabloGrr** [comment]
- Verifica que la imagen no pese más de 10mb.
- Utiliza formatos de imagen (PNG, JPG)
- Sube el archivo de 1 en 1.
- Prueba borrando caché o navegación privada.
- No olvides verificar que estas con sesión iniciada en el correo de launchx / inovaccion

Envié 30 minutos después de las 12 y el formulario seguía habilitado y funcional.
